packages/test/test-service-load/src/runner.ts

```diff
@@ -108,9 +108,27 @@ async function main() {
 	// https://v8.dev/docs/stack-trace-api
 	Error.stackTraceLimit = Infinity;
 
-	process.on("uncaughtExceptionMonitor", (error, origin) => {
+	// We need to install an uncaughtException monitor to prevent Node from killing our process
+	// for not handling exceptions, otherwise we don't get a chance to flush the logger and will
+	// simply lose the telemetry that led up to the uncaught exception.
+	process.on("uncaughtException", (error, origin) => {
```

Comment:
this change will lose coverage for unhandledRejection's. uncaughtExceptionMonitor is called for both unhandledRejection and uncaughtException